test/specs/ReframeService.spec.js

```diff
@@ -0,0 +1,46 @@
+describe('ga_reframe_service', function() {
+  var $rootScope, $httpBackend, gaReframe, lv03tolv95Url, lv95tolv03Url;
+
+  var buildUrl = function(baseUrl, coords) {
+    return baseUrl + '?easting=' + coords[0] + '&northing=' + coords[1];
+  };
+
+  describe('gaReframeService', function() {
+
+    beforeEach(function() {
+      inject(function($injector, gaGlobalOptions) {
+        $rootScope = $injector.get('$rootScope');
+        $httpBackend = $injector.get('$httpBackend');
+        gaReframe = $injector.get('gaReframe');
+        lv03tolv95Url = gaGlobalOptions.lv03tolv95Url;
+        lv95tolv03Url = gaGlobalOptions.lv95tolv03Url;
+      });
+    });
+
```

Comment:
You can also add a test if the request is successful to be sure the godd value is returned.